src/components/layout/Header.tsx

```diff
@@ -63,11 +63,16 @@ const Wrapper = styled.header`
     margin-top: 0;
     color: #e0aacf;
   }
-`
 
-function connectWallet(): void {
-  alert('TODO: Connect to Wallet')
-}
+  a${Wallet} {
```

Comment:
Why specify `a` tag?
If anything you are breaking compatibility with usage as `<Wallet as="button"/>` if we ever needed that

Reply:
Fixed, I think I had an issue with the specificity, but it was solved and that `a` was not removed